docs/installation.md

```diff
@@ -30,6 +30,12 @@ The easiest way to install and run Freqtrade is to clone the bot Github reposito
 !!! Warning "Up-to-date clock"
     The clock on the system running the bot must be accurate, synchronized to a NTP server frequently enough to avoid problems with communication to the exchanges.
 
+!!! Error: Running setup.py install for gym did not run successfully.
```

Comment:
Hah the docs check got this (not me)
the correct syntax is without a ":" 

```suggestion
!!! Error Running setup.py install for gym did not run successfully.
```

Reply:
Nice catch by the ci